tests/scripts/test-ci.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -ex
+
+function push() {
+    arm64_Master_Image=$(docker images --format "{{.Repository}}:{{.Tag}}" | grep "arm64:latest")
```

Comment:
Grep will return multiple items without returning an error. We should consider how important it might be to check that we are only getting a single result. It might not be necessary since we are using `set -x`, but I would lean towards looking for the error so we don't accidentally push accidentally malformed image tags in unforeseen corner cases.